bedrock/firefox/templates/firefox/channel/android.html

```diff
@@ -45,6 +45,9 @@
         <a href="{{ url('privacy.notices.firefox') + '#pre-release' }}">{{ ftl('ui-learn-more') }}</a>
       {% endif %}
     </p>
+    <p>
+      {{ firefox-channel-beta-update-one-or-more-times }}
+    </p>
```

Comment:
This should be in the Firefox Nightly section of the page, not the Firefox Beta section. 

It also throws an error as the syntax is incorrect. Please use the following:

```
{% if ftl_has_messages('firefox-channel-beta-update-one-or-more-times') %}
  <p>
    {{ ftl('firefox-channel-beta-update-one-or-more-times') }}
  </p>
{% endif %}
```